wagtail_localize/views/submit_translations.py

```diff
@@ -222,7 +222,16 @@ def _walk(current_page):
         # TODO: Button that links to page in translations report when we have it
         messages.success(self.request, self.get_success_message(locales))
 
-        return redirect(self.get_success_url() or self.get_default_success_url())
+        single_translated_object = None
+        if len(form.cleaned_data["locales"]) == 1:
```

Comment:
nitpick: we have a `if len(form.cleaned_data["locales"]) == 1:` check just above which preps the locale message, so perhaps merge the two?

Reply:
Good call. I've merged them now!